From 9ce2293715d182f13c3f45f957651bf823b7f16f Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 8 Oct 2020 09:16:47 -0500
Subject: [PATCH 64/71] arm64: dts: imx8mn: Adjust 1.2GHz OPP Voltage to OD
 mode

According to latest datasheet Rev.0, 10/2019, there is restriction
as below:

"If VDD_SOC/GPU/DDR = 0.95V, then VDD_ARM must be >= 0.95V."

As by default SoC is running at OD mode(VDD_SOC = 0.95V), so
VDD_ARM 1.2GHz OPP's voltage should be increased to 0.95V.

According to the regulator tables, our VDD_ARM is .85 while
VDD_SOC is .95V, so we need to increase this.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
index 9926bbd0c275..7ff52357180f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mn.dtsi
@@ -1288,7 +1288,7 @@
 		/* kHz    uV */
 		1500000 1000000
 		1400000 950000
-		1200000 850000
+		1200000 950000
 	>;
 	clocks = <&clk IMX8MN_CLK_A53_DIV>, <&clk IMX8MN_CLK_A53_SRC>,
 		<&clk IMX8MN_ARM_PLL>, <&clk IMX8MN_ARM_PLL_OUT>,
-- 
2.17.1

